### PR TITLE
Release-1.4.4 Docs for Patch Release

### PR DESCRIPTION
**Pull request by bryantidd**

### CODE REVIEW

**Review by stevshap — APPROVED** [review]

File: content/en/docs/setup/prereqs.md [hunk 1]
```diff
@@ -26,12 +26,17 @@ You can install Verrazzano on the following Kubernetes versions.
...
 |------------|--------------|----------------------|---------------------------|------------------------|-------------------------|
-| 1.4        | 2022-09-30   | 1.4.3                | 2023-02-06                | 1.21, 1.22, 1.23, 1.24 |                         |
+| 1.4        | 2022-09-30   | 1.4.4                | 2023-03-15                | 1.21, 1.22, 1.23, 1.24 |                         |
 | 1.3        | 2022-05-24   | 1.3.8                | 2022-11-17                | 1.21, 1.22, 1.23       |                         |
 | 1.2        | 2022-03-14   | 1.2.2                | 2022-05-10                | 1.19, 1.20, 1.21       | 2022-11-30              |
 | 1.1        | 2021-12-16   | 1.1.2                | 2022-03-09                | 1.19, 1.20, 1.21       | 2022-09-30              |
 | 1.0        | 2021-08-02   | 1.0.4                | 2021-12-20                | 1.18, 1.19, 1.20       | 2022-06-30              |
 
+*_End of error correction for Verrazzano releases._<br>
+**_Projected date. Actual date will be determined when the next minor or major release is available._
```
Comment (rosemarymarano) [inline content/en/docs/setup/prereqs.md:36]: Remove this footnote.

File: content/en/docs/setup/prereqs.md [hunk 2]
```diff
@@ -26,12 +26,17 @@ You can install Verrazzano on the following Kubernetes versions.
 
 | Verrazzano | Release Date | Latest Patch Release | Latest Patch Release Date | Kubernetes Versions    | End of Error Correction |
 |------------|--------------|----------------------|---------------------------|------------------------|-------------------------|
-| 1.4        | 2022-09-30   | 1.4.3                | 2023-02-06                | 1.21, 1.22, 1.23, 1.24 |                         |
+| 1.4        | 2022-09-30   | 1.4.4                | 2023-03-15                | 1.21, 1.22, 1.23, 1.24 |                         |
 | 1.3        | 2022-05-24   | 1.3.8                | 2022-11-17                | 1.21, 1.22, 1.23       |                         |
```
Comment (rosemarymarano) [inline content/en/docs/setup/prereqs.md:30]: To VZ 1.3, add this date for end of error correction: 2023-05-31

File: content/en/docs/setup/prereqs.md [hunk 3]
```diff
@@ -26,12 +26,17 @@ You can install Verrazzano on the following Kubernetes versions.
 
 | Verrazzano | Release Date | Latest Patch Release | Latest Patch Release Date | Kubernetes Versions    | End of Error Correction |
 |------------|--------------|----------------------|---------------------------|------------------------|-------------------------|
-| 1.4        | 2022-09-30   | 1.4.3                | 2023-02-06                | 1.21, 1.22, 1.23, 1.24 |                         |
+| 1.4        | 2022-09-30   | 1.4.4                | 2023-03-15                | 1.21, 1.22, 1.23, 1.24 |                         |
```
Comment (rosemarymarano) [inline content/en/docs/setup/prereqs.md:29]: To VZ 1.4, add this date for end of error correction: 2023-10-31

**Review by rosemarymarano — COMMENTED** [review]
WebLogic Monitoring Exporter is not listed as a component, yet we have it in the RNs as a component version update. Maybe it was added in VZ1.4.3 but never added to the components list.

**Review by rosemarymarano — APPROVED** [review]

**Pull request merged**